lib/screens/home/widget/task_tile.dart

```diff
@@ -79,22 +73,33 @@ class _FiAs3AmnaTaskTileState extends State<FiAs3AmnaTaskTile> {
             ),
             Spaces.w25,
             Flexible(
-              child: Text(
-                widget.text,
-                style: TextStyle(
-                  decoration: widget.hasCompleted
-                      ? TextDecoration.lineThrough
-                      : TextDecoration.none,
-                  decorationColor: CustomColors.yellow,
-                  decorationThickness: 2.0,
-                  color: widget.hasCompleted
-                      ? CustomColors.lightGrey
-                      : CustomColors.textColor,
-                  fontSize: 18,
-                  fontWeight: FontWeight.bold,
+              child: Container(
+                alignment: Alignment.topLeft,
+                child: Text(
+                  text,
+                  style: TextStyle(
+                    decoration: hasCompleted
```

Comment:
being completed or not isn't the property of widget itself, it should be the property of task object, you can use make a task model and use that here instead